src/template.js

```diff
@@ -5,7 +5,9 @@ var context = {
   formats:    [],
   format_map: {},
   truncate:   util.truncate,
-  pad:        util.pad
+  pad:        util.pad,
+  monthName:  format.monthName,
```

Comment:
There is probably a better way to access these functions than adding them to the context.